RELEASE_NOTES.md

```diff
@@ -1,3 +1,103 @@
+## Cookstyle 5.6
+
+### 12 New Chef Cops
+
+#### ChefDeprecations/WindowsTaskChangeAction
+
+The `ChefDeprecations/WindowsTaskChangeAction` cop detects the `windows_task` resource used with the legacy `:change` action. When the `windows_task` resource was moved into Chef Infra Client 13, the `:change` action was merged into the `:create` action. Mot users set the `:change` action in conjunction with `:create` and `:change` can now be removed.
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
+#### ChefDeprecations/RecipeMetadata
+
+The `ChefDeprecations/RecipeMetadata` cop detects the usage of the `recipe` method in metadata.rb. This method was never used by Chef Infra and it has been deprecated. User facing documentation such as recipe descriptions should instead be placed in the cookbook's README.md file.
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
+#### ChefStyle/UsePlatformHelpers
+
+The `ChefStyle/UsePlatformHelpers` cop detects the usage of `node['platform] == 'foo'` or `node['platform_family] == 'bar'` and instead suggests using `platform?('foo')` or `platform_family?('bar')`. These platform helpers help simplify platform check logic, especially when checking for multiple platforms as they accept more than one platform: `platform?('foo', 'bar')
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
+#### ChefModernize/CronManageResource
+
+The `ChefModernize/CronManageResource` cop detects the usage of the `cron_manage` resource, which was renamed to  `cron_access` when it was included in Chef Infra Client 14.4.
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
+#### ChefModernize/UsesZypperRepo
+
+The `ChefModernize/UsesZypperRepo` cop detects the usage of the `zypper_repo` resource, which was renamed to  `zypper_repository` when it was included in Chef Infra Client 13.3.
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
+#### ChefModernize/DependsOnZypperCookbook
+
+The `ChefModernize/DependsOnZypperCookbook` cop detects a cookbook that depends on the `zypper` cookbook. Chef Infra Client 13.3 and later include the `zypper_repository`, which was previously only available in the `zypper` cookbook.
```

Comment:
If you could put a modifying word after `zypper_repository`, such as "resource" or other, it would improve the sentence.

Reply:
oh yep that should be resource. I'll add that

Reply:
Fixed